packages/hydra-indexer/src/entities/SubstrateEventEntity.ts

```diff
@@ -57,7 +68,7 @@ export class SubstrateEventEntity extends AbstractWarthogModel {
 
   // PG int type size is not large enough
   @Column('numeric', { transformer: new NumericTransformer() })
```

Comment:
It would be better to set the column type to `bigint` and get rid of the transformer right?

Reply:
It's fixed by commit https://github.com/Joystream/hydra/pull/356/commits/0ea5f0e01387affd4e8f7a5bea053fabe4813ed6